docs/maui/essentials/apptheme-resources.md

```diff
@@ -0,0 +1,119 @@
+---
+title: AppTheme Resources - .NET MAUI Community Toolkit
+author: jfversluis
+description: With AppThemeResource and AppThemeColor you can create theme aware resources for your application that automatically update when the device theme updates.
+ms.date: 07/13/2023
+---
+
+# AppTheme Resources
+
+With `AppThemeResource` and `AppThemeColor` you can create theme-aware resources for your application that automatically update when the device theme updates.
+
+The `AppThemeResource` and `AppThemeColor` objects are theme-aware resources that will make it easier to work with colors, images, and other resources that need to change depending on the app's current theme.
+These objects build upon the concepts of the [`AppThemeBinding`](/dotnet/maui/user-interface/system-theme-changes) that is available in .NET MAUI, and will make it easier to work with these types of resources in a [`ResourceDictionary`](/dotnet/maui/fundamentals/resource-dictionaries).
+
+Because of this, you should typically use these APIs through the `ThemeResource` markup extension in XAML.
+
+## Syntax
+
+### Including the XAML namespace
+
+[!INCLUDE [XAML usage guidance](../includes/xaml-usage.md)]
+
+#### AppThemeResource
+
+The `AppThemeResource` is a generic theme-aware object that allows you to set any `object` for the `Light`, `Dark` and `Default` properties. Because `AppThemeResource` is not strongly-typed, at runtime the values for each property will be evaluated and casted.
```

Comment:
```suggestion
The `AppThemeObject` is a generic theme-aware object that allows you to set any `object` for the `Light`, `Dark` and `Default` properties. Because `AppThemeObject` is not strongly-typed, at runtime the values for each property will be evaluated and casted.
```